blockchain/difficulty.go

```diff
@@ -286,7 +287,8 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 
 	// Limit the amount of adjustment that can occur to the previous
 	// difficulty.
-	actualTimespan := lastNode.timestamp.UnixNano() - firstNode.timestamp.UnixNano()
```

Comment:
this removed code actually looks pretty scary considering it could overflow if the nanoseconds exceeds the max int64 value.  We sure this isn't a fork? (were we even correct before?)